### DISCUSSION TITLE
"libpag/web" environment - "wasm" remains in memory after page refresh.

### DISCUSSION THREAD

**Discussion by tobenpark** [Tencent/libpag#1980] [Q&A] [answered]
My question is the same as the title

After loading libpag.wasm module
How do I use wasm to release from memory?

Thank you.

**Comment by domchen** [comment, marked as answer]
Wasm instance objects are automatically garbage-collected, like with everything else on the JS heap. Thus, if you wish to release the wasm instance, you can delete all the global variables with the PAG prefixed. This should do the work.
